Makefile

```diff
@@ -218,13 +221,33 @@ kind-bootstrap-cluster: kind-create-cluster install-crds webhook kind-deploy-con
 .PHONY: kind-bootstrap-cluster-dev
 kind-bootstrap-cluster-dev: kind-create-cluster install-crds install-resources
 
-webhook:
-	-kubectl create ns $(KIND_NAMESPACE)
-	@echo installing cert-manager
+cert-manager:
+	@echo Installing cert-manager
 	kubectl apply -f https://github.com/cert-manager/cert-manager/releases/download/v1.12.0/cert-manager.yaml
-	@echo "wait until pods are up"
+	@echo "Waiting until the pods are up"
 	kubectl wait deployment -n cert-manager cert-manager --for condition=Available=True --timeout=180s
 	kubectl wait --for=condition=Ready pod -l app.kubernetes.io/instance=cert-manager -n cert-manager --timeout=180s 
+
+postgres: cert-manager
+	@echo "Installing Postgres"
+	-kubectl create ns $(KIND_NAMESPACE)
+	sed 's/open-cluster-management/$(KIND_NAMESPACE)/g' build/kind/postgres.yaml | kubectl apply --timeout=180s -f-
+
+	@echo "Waiting until the pods are up"
+	@sleep 3
```

Comment:
Why we need this? we have `wait` command

Reply:
It's a hack in case you run `make postgres` again with some changes in the YAML. If you go too fast to the `kubectl wait` command, the Deployment hasn't started a new rollout yet so the `kubectl wait` command uses the old pod.